pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/pubsub/MercuryOrderPublisher.java

```diff
@@ -99,6 +91,15 @@ public void onSuccess(String messageId) {
         }
     }
 
+    private String createMercuryUniqueOrderId() {
+        String orderNumber = NanoIdUtil.getNanoId("1234567890QWERTYUIOPASDFGHJKLZXCVBNM", 20);
```

Comment:
What's the volume of orders here?  
How often will we have ID collision?
Are these for human consumption?  If not, how about a UUID?

Reply:
I think the volume is ~ 10 orders every month or so